src/client/packaging/pypi/pyproject.toml

```diff
@@ -0,0 +1,118 @@
+# Guide (user-friendly):
+# https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
+
+# Specification (technical, formal):
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
+
+
+# Choosing a build backend:
+# https://packaging.python.org/en/latest/tutorials/packaging-projects/#choosing-a-build-backend
+[build-system]
+# A list of packages that are needed to build your package:
+requires = ["setuptools>=61.0.0"] # REQUIRED if [build-system] table is used
```

Comment:
what does that version get us?  the sample/template `pyproject.toml` [doesnt specify it](https://github.com/pypa/sampleproject/blob/a1f00056ebc65d17ae69c5aba324ef4f0382138a/pyproject.toml#L12)

Reply:
Ah yea, it's not necessary here, I'll remove it. It's needed to support the dynamic version feature (see the callout box in bullet 1 [here](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/) "As of the release of setuptools 61.0.0...").